plugins/woocommerce/src/Blocks/BlockTypes/AbstractProductGrid.php

```diff
@@ -666,6 +666,18 @@ protected function get_add_to_cart( $product ) {
 			$attributes['class'] .= ' ajax_add_to_cart';
 		}
 
+		/**
+		 * Filter to add additional attributes to the HTML code of the generated add to cart button.
```

Comment:
The filter is not to add but can also modify the existing attribute, so I think we should use a broader term here.

Reply:
I changed the filter description in a new commit.